stable/external-dns/README.md

```diff
@@ -126,8 +129,6 @@ The following table lists the configurable parameters of the external-dns chart
 | `rbac.create`                      | Wether to create & use RBAC resources or not                                                             | `false`                                                  |
 | `rbac.serviceAccountName`          | ServiceAccount (ignored if rbac.create == true)                                                          | `default`                                                |
 | `rbac.apiVersion`                  | Version of the RBAC API                                                                                  | `v1beta1`                                                |
-| `securityContext.fsGroup`          | Group ID for the container                                                                               | `1001`                                                   |
```

Comment:
I would add the `securityContext` parameter with default value to `{}` as it's set in the `values.yaml`

Reply:
done